src/cli/near.rs

```diff
@@ -227,13 +304,124 @@ pub async fn execute_command<T: AsRef<str>>(
                 };
             }
             ReadCommand::GetAuroraErc20 { nep_141_account } => {
-                println!("{:?}", client.get_erc20_from_nep141(&nep_141_account).await);
+                println!("{:?}", client.get_erc20_from_nep141(&nep_141_account).await?);
             }
             ReadCommand::GetEngineBridgeProver => {
-                println!("{:?}", client.get_bridge_prover().await);
+                println!("{:?}", client.get_bridge_prover().await?);
+            }
+            ReadCommand::GetChainId => {
+                let chain_id = {
+                    let result = client.near_view_call("get_chain_id".into(), vec![]).await?;
+                    U256::from_big_endian(&result.result).low_u64()
+                };
+                println!("{:?}", chain_id);
             }
+            ReadCommand::GetUpgradeIndex => {
+                let upgrade_index = {
+                    let result = client
+                        .near_view_call("get_upgrade_index".into(), vec![])
+                        .await?;
+                    U256::from_big_endian(&result.result).low_u64()
+                };
+                println!("{:?}", upgrade_index);
+            }
+            ReadCommand::GetBlockHash {
+                block_number
+            } => {
+                let height_serialized: u128 =  block_number.parse::<u128>().unwrap();
+                let block_hash = {
+                    let result = client
+                        .near_view_call("get_block_hash".into(), height_serialized.to_le_bytes().to_vec())
+                        .await?;
+                    println!("{:?}", result);
```

Comment:
Unneeded print?

Reply:
I think I was using it for debugging, removing it now.